test/integration/krane_test.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+require 'integration_test_helper'
+
+class KraneTest < KubernetesDeploy::IntegrationTest
+  def test_restart_black_box
+    assert_deploy_success(deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "web.yml.erb", "redis.yml"]))
+    refute(fetch_restarted_at("web"), "no RESTARTED_AT env on fresh deployment")
```

Comment:
Bit of a nitpick but the second argument to `refute`/`assert` should be the failure message. So like, if `refute fetch_restarted_at("web")` _failed_ it would mean `RESTARTED_AT` was found on a fresh deployment somehow and that would be your error message.

Reply:
Yep, that's what I get for copy/pasting without thinking. Fixed.